samples/wifi/radio_test/src/nrf_wifi_radio_ficr_shell.c

```diff
@@ -298,13 +454,39 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 			  nrf_wifi_radio_test_otp_read_params,
 			  1,
 			  0),
+	SHELL_CMD_ARG(otp_read_retrim_version,
+			  NULL,
+			  "Read Retrim Version",
+			  nrf_wifi_radio_test_otp_read_retrim_version,
+			  1,
+			  0),
+	SHELL_CMD_ARG(otp_read_retrim_params,
+			  NULL,
+			  "Read Retrim Params",
+			  nrf_wifi_radio_test_otp_read_retrim_params,
+			  1,
+			  0),
 	SHELL_CMD_ARG(otp_write_params,
 			  NULL,
 			  "Write OTP Params\n"
 			  "otp_write_params <addr offset> [arg1] [arg2]...[argN]",
 			  nrf_wifi_radio_test_otp_write_params,
 			  2,
 			  16),
+	SHELL_CMD_ARG(otp_write_retrim_version,
+			  NULL,
+			  "Write OTP Retrim Version\n"
+			  "otp_write_retrim_version <retrim version>",
+			  nrf_wifi_radio_test_otp_write_retrim_version,
+			  2,
+			  0),
+	SHELL_CMD_ARG(otp_write_retrim_params,
+			  NULL,
+			  "Write OTP Retrim Params\n"
```

Comment:
`retrim parameters` instead of `Retrim Params` ?

Reply:
Hi Sachin, it's to have consistency with older sub commands.

	SHELL_CMD_ARG(otp_write_params,
			  NULL,
			  "Write OTP Params\n"
			  "otp_write_params <addr offset> [arg1] [arg2]...[argN]",
			  nrf_wifi_radio_test_otp_write_params,
			  2,
			  16),

Reply:
Ideally we should try to fix the older subcommands too (as a separate commit) :)..but I understand the urgency here.